env/work/srw_python/srwlib.py

```diff
@@ -3409,11 +3633,64 @@ def srwl_uti_read_intens_ascii(_file_path, _num_type='f'):
     f.close()
     return array(_num_type, arInt), resMesh
 
-#**********************Auxiliary function to write auxiliary/debugging information to an ASCII file:
-def srwl_uti_save_text(_text, _file_path):
-    f = open(_file_path, 'w')
-    f.write(_text + '\n')
-    f.close()
```

Comment:
Rewrote in more modern style with more flexibility (appending instead of rewriting by default). See https://github.com/ochubar/SRW/pull/7/files#diff-d8ea76648d23e6da5b2a0010abbede1dR3691.